src/components/Tooltip/EducationalTooltip/BaseEducationalTooltip.tsx

```diff
@@ -28,6 +29,8 @@ function BaseEducationalTooltip({children, shouldRender = false, shouldHideOnNav
     const navigator = useContext(NavigationContext);
     const insets = useSafeAreaInsets();
 
+    const isNative = getPlatform() === CONST.PLATFORM.IOS || getPlatform() === CONST.PLATFORM.ANDROID;
```

Comment:
https://github.com/expensify/app?tab=readme-ov-file#platform-specific-file-extensions

Let's use platform-specific files instead of calling `getPlatform()`
And could you please also add comments explaining why we use different methods on web/native?